src/BizHawk.Client.Common/movie/tasproj/TasMovie.Editing.cs

```diff
@@ -114,7 +114,8 @@ public void RemoveFrames(ICollection<int> frames)
 			{
 				// Separate the given frames into contiguous blocks
 				// and process each block independently
-				List<int> framesToDelete = frames.OrderBy(fr => fr).ToList();
+				List<int> framesToDelete = frames.Where(fr => fr >= 0 && fr < InputLogLength)

```

Comment:
super nitpick but can you put the LINQ each on separate lines?
framesToDelete = frames
    .Where()
    .OrderBy()
   .ToList()